test/test_root_hit.jl

```diff
@@ -0,0 +1,115 @@
+@testset "RootHitFiles" begin
```

Comment:
Should be `"@testset "RootHitFile" begin`

Reply:
Sure. This is a vestige of it being its own package